task_api/spec/requests/api/labels_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
+
+describe 'Labels APIのテスト' do
+  before do
+    FactoryBot.create(:task, :task1)
+    FactoryBot.create(:task, :task2)
+    FactoryBot.create(:task, :task3)
+
+    FactoryBot.create(:label, :label1)
+    FactoryBot.create(:label, :label2)
+    FactoryBot.create(:label, :label3)
+  end
+
+  context '新しいLabelの作成' do
+    it '成功事例' do
+      new_label = { name: 'Very Fun' }
+
+      post '/api/tasks/1/labels', params: new_label
+      body = JSON.parse(response.body)
+      inserted_label = Task.find(1).labels.last
+
+      expect(response.status).to eq(200)
+      expect(body['message'].class).to eq(String)
```

Comment:
型のチェックなら `instance_of` か `kind_of` でもOKです。
（型のチェックまでいるかな？）

```suggestion
      expect(body['message']).to be_instance_of String
```

Reply:
型チェックいらないかもですね．．．
せっかく`instance_of`を教えて頂いたので，型チェックはtoo muchかもしれませんが，備忘録も兼ねて入れておきたいと思います